CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ librdkafka v2.3.1 is a maintenance release:
    and updating the topic id corresponding to the partition name (#4660)
  * Fixes to metadata cache expiration, metadata refresh interruption and
    to avoid usage of stale metadata (#4660).
+ * Fix to main loop timeout calculation leading to a tight loop for a
+   max period of 1 ms (#4671).
```

Comment:
Let's use something easier to understand than `tight loop`. The later statement is more comprehendible than this.